currency_rate_update/models/res_currency_rate_provider.py

```diff
@@ -318,7 +324,9 @@ def _scheduled_update(self):
                     provider.next_run - provider._get_next_run_period()
                 )
                 date_to = provider.next_run
-                provider._update(date_from, date_to, newest_only=True)
+                close_date = provider._get_close_date()
+                if not close_date or (close_date and datetime.now().hour >= close_date):
```

Comment:
You must compare the close_date only for fetching data from today, not from any date interval.

Reply:
> Check also Travis status

Travis solved